app/src/main/java/protect/card_locker/MainActivity.java

```diff
@@ -292,6 +292,15 @@ public void onTabReselected(TabLayout.Tab tab) {
 
         mNoMatchingCardsText.setOnTouchListener(gestureTouchListener);
         mCardList.setOnTouchListener(gestureTouchListener);
+        mCardList.addOnScrollListener(new RecyclerView.OnScrollListener() {
+            @Override
+            public void onScrolled(@NonNull RecyclerView recyclerView, int dx, int dy) {
+                super.onScrolled(recyclerView, dx, dy);
+
+                showArchiveTextIfArchived();
+            }
+        });
+        mCardList.getViewTreeObserver().addOnGlobalLayoutListener(() -> showArchiveTextIfArchived());
```

Comment:
This doesn't work.

If the current group should show archive text, it doesn't appear until the activity lost and regained focus.